src/styles/styles.css

```diff
@@ -0,0 +1,316 @@
+@charset "UTF-8";
+
+/* VARIABLES HEADER */
+
+/* VARIABLES CARD */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+/* #region HEADER */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.header__nav {
+  padding: 0;
+}
+
+.header__nav-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.header__nav-item {
+  margin-right: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.header__nav-item:last-child {
+  margin-right: 0;
+}
+
+.header__nav-item__link {
+  color: #060b35;
+  text-decoration: none;
+  display: block;
+  line-height: 60px;
+}
+
+.header__nav-item__link:hover,
+.header__nav-item__link:active {
+  color: #00acdc;
+}
+
+.header__nav-item__link:active {
+  text-decoration: underline;
+}
+
+.header__nav-item__link.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.header__nav-item__link.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 2px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.header__logo-link,
+.header__logo-link img {
+  width: 40px;
+  height: auto;
+  display: block;
+}
+
+/* #endregion HEADER */
+
+/* #region CATALOG */
+
+.catalog {
+  display: grid;
+  width: 944px;
+  grid-template-columns: repeat(4, 200px); /* 4 колонки по 200px */
+  justify-content: center;
+  gap: 46px 48px;
+  margin: 50px auto;
+}
```

Comment:
The grid properties could be improved. The `grid-template-columns: repeat(4, 200px);` property could be replaced with `grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));` to create a responsive layout. The `width: 944px;` property could be removed as it's not needed with the improved grid properties.